src/OpenDiffix.Core.Tests/QueryEngine.Tests.fs

```diff
@@ -187,11 +187,70 @@ type Tests(db: DBFixture) =
   [<Fact>]
   let ``query 14 - avg_noise parity`` () =
     let expected = runQuery "SELECT sum_noise(age) / count(age) as avg_noise FROM customers_small GROUP BY city"
-
     let queryResult = runQuery "SELECT avg_noise(age) FROM customers_small GROUP BY city"
 
     queryResult |> should equal expected
 
+  let ``query 15 - count histogram`` () =
```

Comment:
this is my copy pasta, but maybe you could fix this - I've put in `query 14` twice above, so yours is 16